### PR TITLE
Fix smarty notices on localization page

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
civicrm/admin/setting/localization?reset=1

Before
----------------------------------------

image

After
----------------------------------------
image

Technical Details
----------------------------------------
Remaining one is in a different tpl

Comments
----------------------------------------

### CODE REVIEW

File: templates/CRM/Admin/Form/Setting/Localization.tpl [hunk 1]
```diff
@@ -37,7 +37,7 @@
                 <span class="description">{$settings_fields.inheritLocale.description}</span>
               </td>
             </tr>
-          {if empty($form.languageLimit)}
+          {if array_key_exists('languageLimit', $form)}
```
Comment (demeritcowboy) [inline templates/CRM/Admin/Form/Setting/Localization.tpl:40]: This looks !(not wrong).

But actually it looks like it was also previously checking the wrong var - it should be uiLanguages.

Comment (demeritcowboy) [reply templates/CRM/Admin/Form/Setting/Localization.tpl:40]: So isset is problematic but array_key_exists is ok for the smarty default escape mode? Interesting.

Comment (eileenmcnaughton) [reply templates/CRM/Admin/Form/Setting/Localization.tpl:40]: @demeritcowboy yep - I discovered that `$form` is actually assigns as an array to the tpl layer - I can't find where right now

Comment (demeritcowboy) [reply templates/CRM/Admin/Form/Setting/Localization.tpl:40]: Ok so it's changed from a negative to a positive which changes the display logic. Now I'm trying to figure out if uiLanguages is related to languageLimit, i.e. was the original `if` statement intended as display logic or was it a guard against notices.

It's confusing. These fields have the same title.

**Comment by eileenmcnaughton** [comment]
I just rebased the other commit out

File: templates/CRM/Admin/Form/Setting/Localization.tpl (hunk 1 above)
Comment (eileenmcnaughton) [reply templates/CRM/Admin/Form/Setting/Localization.tpl]: ok - it' some weird logic - but I've fixed it so it should go back to no-change

Comment (demeritcowboy) [reply templates/CRM/Admin/Form/Setting/Localization.tpl]: Yeah the original appears to be about display logic - languageLimit only gets set for multilingual, and for single the field is a different field uiLanguages, so the `if`'s are saying only show one.

**Pull request merged**